checkers/blsRatingsChecker.go

```diff
@@ -0,0 +1,92 @@
+package checkers
+
+import (
+	"fmt"
+
+	"multiversx/mvx-keys-monitor/core"
```

Comment:
why this import doesn't contains `github.com` ?

Reply:
renamed. Good catch :+1: